src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -81,6 +83,8 @@ function renderDropdown(action, display, style) {
       onChange={(id) => action(id)}
       options={options?.length === 0 ? [{value: '', label: noOptionsMessage}] : options}
       disabled={!options?.length}
+      tooltip={tooltip}
+      title={tooltip}
```

Comment:
is `title` needed here?

Reply:
You are right. There is no need for it. I took it out.